recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -28,18 +26,28 @@ def _min_cppstd(self):
 
     @property
     def _compilers_minimum_version(self):
-        return {
-            "gcc": "10.2",
-            "Visual Studio": "16.9",
-            "clang": "12",
-            "apple-clang": "13",
-        }
+        if not self.version.startswith("cci.") and  tools.Version(self.version) >= "2022.03.0":
```

Comment:
Maybe something more generic?

```suggestion
        if "2021" not in self.version:
```

If someone tries to add `cci.2022...` this would fail 😢

Reply:
what do you mean exactly ?

Reply:
Normally, there is no reason for such a release to ever exist.
The first cci was made by the original writer of this recipe. I took over and added the newest release
with our proper versioning scheme.

So do we really need to fix this ?
FYI, there is another release planned "soon" and it would be great if this recipe gets stabilized so adding new versions is easier on everyone (us when writing it, conan reviewers when reviewing it).

Thanks a lot for all the comments though :)

Reply:
If it helps to simplify the recipe, I would suggest removing version `cci.20210823` now that the library has proper releases. wdyt?

Reply:
Stupidly, I didnt think about it. Yes, it may solve the issue.